scripts/init.sh

```diff
@@ -39,5 +37,15 @@ sifnoded validate-genesis
 
 
 
+#mkdir -p $DAEMON_HOME/cosmovisor/genesis/bin
```

Comment:
Can we remove this or move it to its own test script?

Reply:
we do need this , to create directories but I think they can be made part of the deployment scripts rather than these shell scripts